cfme/tests/infrastructure/test_pxe_provisioning.py

```diff
@@ -60,10 +60,10 @@ def pytest_generate_tests(metafunc):
 
 @pytest.fixture(scope="function")
 def setup_pxe_servers_vm_prov(pxe_server, pxe_cust_template, provisioning):
-    if not pxe_server.exists:
+    if not pxe_server.exists():
         pxe_server.create()
     pxe_server.set_pxe_image_type(provisioning['pxe_image'], provisioning['pxe_image_type'])
-    if not pxe_cust_template.exists:
+    if not pxe_cust_template.exists():
```

Comment:
someone broke these tests by renaming exists calls to properties but he wasn't anxious about changing pxe code. I just fixed tests. I didn't want to make changes to pxe calls due to sentaku.
However, I'll change them as well if you insist.